Hover animation spams errors on client when using Notification.new

Stack:
```
[ERROR] addons/cfc_notifications/lua/cfc_notifications/client/cl_dnotificationbutton.lua:9: attempt to perform arithmetic on local 'prog' (a nil value)
  1. adjustProg - addons/cfc_notifications/lua/cfc_notifications/client/cl_dnotificationbutton.lua:9
   2. unknown - addons/cfc_notifications/lua/cfc_notifications/client/cl_dnotificationbutton.lua:96
```

--- a/lua/cfc_notifications/client/cl_dnotification.lua
+++ b/lua/cfc_notifications/client/cl_dnotification.lua
@@ -142,8 +142,8 @@ function PANEL:_makeTitleBar()
     if self:GetCloseable() then
         table.insert( btnTexts, { "discard", "OnClose" } )
         if self:GetIgnoreable() then
-            table.insert( btnTexts, { "mute", "OnIgnore", true } )
-            table.insert( btnTexts, { "temporary mute", "OnIgnore" } )
+            table.insert( btnTexts, { "mute", "OnIgnore" } )
+            table.insert( btnTexts, { "never show again", "OnIgnore", true } )
         end
     end
 

--- a/lua/cfc_notifications/client/cl_dnotificationbutton.lua
+++ b/lua/cfc_notifications/client/cl_dnotificationbutton.lua
@@ -93,7 +93,7 @@ function PANEL:Paint( w, h )
 
     -- Hover animation
     -- Adjusted animState (similar to prog) to be sinusoidal
-    local prog = adjustProg( prog )
+    local prog = adjustProg( self.animState )
     local inverseProg = 1 - prog
 
     -- Underline foreground

--- a/lua/cfc_notifications/client/cl_render.lua
+++ b/lua/cfc_notifications/client/cl_render.lua
@@ -266,7 +266,7 @@ local function addNotifHooks( panel, popupID )
                 v.panel:OnClose()
             end
         end
-        self.data.notification:Ignore( self )
+        self.data.notification:Ignore( permanent )
     end
 end
 

